### PR TITLE
support plan replayer for multi sqls

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11317
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Yisaer — APPROVED** [review]

File: sql-plan-replayer.md [hunk 1]
```diff
@@ -57,14 +57,55 @@ plan replayer dump explain select * from t;
...
+```sql
++-----------------------------------------------------------+
+| @@tidb_last_plan_replayer_token                           |
++-----------------------------------------------------------+
+| replayer_Fdamsm3C7ZiPJ-LQqgVjkA==_1663304195885090000.zip |
++-----------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+When there are multiple SQL statements, you can obtain the result of the `PLAN REPLAYER DUMP` execution using a file. The results of multiple SQL statements are separated by `;` in this file.
```
Comment (Oreoxmt) [inline sql-plan-replayer.md:86]:
```suggestion
When there are multiple SQL statements, you can obtain the result of the `PLAN REPLAYER DUMP` execution using a file. Multiple SQL statements are separated by `;` in this file.
```

这里指的应该是 sqls.txt 里面的多条 SQL 语句要以 ; 分割

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**